docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.7"
+services:
+  app:
+    container_name: berth-reservation-admin
+    build: .
+    volumes:
+      - .:/home/node/app
+      - /home/node/app/node_modules
+    working_dir: /home/node/app
+    environment:
+      NODE_ENV: development
```

Comment:
I would rather like to see this set in the Dockerfile as it has a `development` stage.

Reply:
true, I removed this from docker-compose.